pkgs/by-name/li/libspelling/package.nix

```
@@ -44,12 +44,15 @@ stdenv.mkDerivation (finalAttrs: {
   buildInputs = [
     glib
     gtk4
-    gtksourceview5
     enchant
     icu
     libsysprof-capture
   ];
 
+  propagatedBuildInputs = [
+    gtksourceview5
```

Note that it lists `Requires: gio-2.0, gtk4, gtksourceview-5` so for completeness sake, it should be:

```suggestion
    # Listed in `Requires` key of `libspelling-1.pc`
    glib
    gtk4
    gtksourceview5
```

---

done